lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -42,24 +42,24 @@
 }
 
 .forum-nav-browse {
-    box-sizing: border-box;
-    width: 100%;
-    vertical-align: middle;
-    padding: 11px;
-    border: 0;
-    border-radius: 0;
-    background: transparent;
-    box-shadow: none;
+  box-sizing: border-box;
```

Comment:
I was making it consistent with the rest of the file. It looks like the only parts that used four spaces are the new lines you added recently. Since I prefer four spaces too, I'll re-indent the whole file to use four.